pages/apis/graphql/graphql_cookbook.md

```diff
@@ -851,6 +852,30 @@ Query your organization's audit events. Audit events are only available to Enter
   }
 ```
 
+To get all audit events in a given period, use the `occurredAtFrom` and `occurredAtTo` filters like in the following query:
+
+```graphql
+  query getOrganizationAuditEvents{
+    organization(slug:"organization-slug"){
+      auditEvents(first: 500, occurredAtFrom: "2023-01-01T12:00:00.000", occurredAtTo: "2023-01-01T13:00:00.000"){
+        edges{
+          node{
+            type
+            occurredAt
+            actor{
+              name
+            }
+            subject{
+              name
+              type
+            }
+          }
+        }
+      }
+    }
+  }
```

Comment:
I don't think this needs to be indented?

```suggestion
query getOrganizationAuditEvents{
  organization(slug:"organization-slug"){
    auditEvents(first: 500, occurredAtFrom: "2023-01-01T12:00:00.000", occurredAtTo: "2023-01-01T13:00:00.000"){
      edges{
        node{
          type
          occurredAt
          actor{
            name
          }
          subject{
            name
            type
          }
        }
      }
    }
  }
}
```

Reply:
Fixed that extra indentation @mbelton-buildkite 😉 (sitting on me for a bit - apologies!)